### PR TITLE
update favicons

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

File: _themes/cakephp/layout.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 {% endblock -%}
 
 {% block extrahead %}
-<link rel="apple-touch-icon" sizes="180x180" href="{{ pathto('_static/apple-touch-icon.png', 1) }}">
+<link rel="apple-touch-icon" sizes="180x180" href="{{ pathto('/apple-touch-icon.png', 1) }}">
```
Comment (markstory) [inline _themes/cakephp/layout.html:22]: I think files might need to be in the `static` directory to get moved in place by tinker (the tool that builds the bakery).

Comment (Arhell) [reply _themes/cakephp/layout.html:22]: i need this file + .ico file to be in the main directory
https://realfavicongenerator.net/favicon_checker?protocol=https&site=bakery.cakephp.org%2F#.YAbuHugzaMo
Example
now https://bakery.cakephp.org/_static/apple-touch-icon.png
need https://bakery.cakephp.org/apple-touch-icon.png

Comment (markstory) [reply _themes/cakephp/layout.html:22]: Ok. We will need to move that file after the site builds then. I don't remember exactly how to do that with dokku though.

Comment (Arhell) [reply _themes/cakephp/layout.html:22]: Let's try, if we fix anything

Comment (markstory) [reply _themes/cakephp/layout.html:22]: Looks like the favicon is broken now.

Comment (Arhell) [reply _themes/cakephp/layout.html:22]: I checked, you are right.
I will correct it soon

**Pull request merged**